### PR TITLE
2285 - Basic Database Creation x 2

### PR DESCRIPTION
**Pull request by kanjula-archana**
Two separate containers are created for the two Postgres Databases (Accounts & Transactions). Additionally, a PG admin instance is also set up for looking at and accessing the data without installing it in our local.

In order to be able to access the Databases through PgAdmin there are a few manual steps still required. We would need to add the servers we are trying to access to PgAdmin. I have added these steps to the Slack Channel and pinned the same.

Please take a look and let me know if any updates are required.

### CODE REVIEW

**Comment by kanjula-archana** [comment]
This is related to [AB#2285](https://deloittedigitalat.visualstudio.com/core-banking/_boards/board/t/core-banking%20Team/Stories/?workitem=2285)

File: docker-compose.yml [hunk 1]
```diff
@@ -0,0 +1,46 @@
+version: '3'
```
Comment (aberke220) [inline docker-compose.yml:1]: Could you please move this file into the `/database` dir? Currently it is being used exclusively for database setup. Thanks!

Comment (dominicboeck) [reply docker-compose.yml:1]: Please also adjust the version as seen here: https://docs.docker.com/compose/compose-file/compose-versioning/#version-3

Currently it's 3.0, but there is already 3.8.

File: database/Transactions.sql [hunk 2]
```diff
@@ -0,0 +1,37 @@
+CREATE TABLE TRANSACTION
+(
+TRANS_ID BIGINT GENERATED ALWAYS AS IDENTITY (START WITH 10000),
+ACCOUNT_ID VARCHAR (20),
+ACCOUNT_IBAN VARCHAR (20),
+TYPES VARCHAR (20),
+AMOUNT FLOAT,
```
Comment (aberke220) [inline database/Transactions.sql:7]: Is `FLOAT` the right type to store monetary values? In Java we usually define it with `BigDecimal`, not `float` due to the possible loss of precision with arithmetic operations. 
I am unfamiliar with recommended type for storing in Postgres though.

File: docker-compose.yml (hunk 1 above)
Comment (kanjula-archana) [reply docker-compose.yml]: Updated the location & version of the docker file.

File: database/Transactions.sql (hunk 2 above)
Comment (kanjula-archana) [reply database/Transactions.sql]: Updated the datatype to decimal(19,2) as the precision. We can updated it according to data so it should be more suitable for money.

**Review by aberke220 — APPROVED** [review]
Thanks for the updates!

File: database/docker-compose.yml [hunk 3]
```diff
@@ -0,0 +1,46 @@
+version: '3.8'
+
+services:
+
+  postgres:
```
Comment (theqajedi) [inline database/docker-compose.yml:5]: can you please give the actual name?
like accountdb similarly for the below one as well?

Comment (kanjula-archana) [reply database/docker-compose.yml:5]: Sure, I have updated the names to Accountsdb & Transactionsdb.

**Pull request merged**